.github/workflows/test-action.yml

```diff
@@ -0,0 +1,43 @@
+name: Test GitHub Action
+on: [pull_request]
+
+jobs:
+  run:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Actions Repository
+      uses: actions/checkout@v2
+
+    - name: Test force pass with mistakes
+      uses: ./
+      with: 
+        files: ./src/file.rs
```

Comment:
We should generate this file so this doesn't stop testing things for us if I refactor `file.rs`

Maybe a `test/typo.txt` file.  I could then have `src/file` and other places source that as well and we could then have it generally ignored, allowing running `typos` on `typos`.

Reply:
What about generating the file on the fly like the other tests? It makes it easy to have tiny differences. And if you decide to use the text in other places it could be written to file!

Reply:
I think thats fine.  I was just adding another option to try to isolate these test cases so its easier to run typos on itself.

Reply:
Great!